docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -1,8 +1,13 @@
 # Clearwater Configuration Options Reference
 
-This document describes all the Clearwater configuration options that can be set in /etc/clearwater/shared_config, /etc/clearwater/local_config or /etc/clearwater/user_settings.
+This document describes all the Clearwater configuration options that can be set in `/etc/clearwater/shared_config`, `/etc/clearwater/local_config` or `/etc/clearwater/user_settings`.
 
-You should follow [this process](Modifying_Clearwater_settings.md) when changing most of these settings. However for settings in the "Local settings" or "User settings" you should:
+At a high level, these files contain the following types of configuration options:
+* `shared_config` - This file holds settings that are common across the entire deployment.
```

Comment:
It may worth stating at this point that changes to this file should and will be synchronised across the whole deployment. 
'Changes should by synchronised using <script>, and will trigger a rolling restart of the deployment where necessary. See more on this [here]'